content/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more/index.md

```diff
@@ -87,7 +87,7 @@ const kibana = new helm.v2.Chart("kibana", {
 
 Check out the [Kubernetes
 overview]({{< relref "/docs/get-started/kubernetes" >}}) docs, the [API
-documentation]({{< relref "/docs/reference/pkg/nodejs/pulumi/kubernetes" >}})
+documentation]({{< relref "/docs/reference/pkg/nodejs/pulumi/kubernetes" TODO NEW K8s doc link>}})
```

Comment:
Kubernetes is now available here: `/docs/reference/pkg/kubernetes`. It was merged last night.